components/TextFields/tests/snapshot/MDCTextControlSnapshotTestHelpers.m

```diff
@@ -48,6 +73,15 @@ + (void)forceLayoutOfTextControl:(UIView<MDCTextControl> *)textControl {
   [textControl sizeToFit];
   [textControl setNeedsLayout];
   [textControl layoutIfNeeded];
+  UIView *container = [textControl superview];
+  CGRect textControlFrame = textControl.frame;
+  CGFloat textControlWidth = CGRectGetWidth(textControlFrame);
+  CGFloat textControlHeight = CGRectGetHeight(textControlFrame);
+  CGRect newContainerFrame = CGRectMake(0, 0, textControlWidth + 30, textControlHeight + 30);
```

Comment:
what is the 30 and 15 paddings used for?

Reply:
15 is just an arbitrary padding. I am making the code more clear.